crates/model/src/signature.rs

```diff
@@ -12,14 +12,17 @@ use web3::{
 #[serde(rename_all = "lowercase")]
 pub enum SigningScheme {
     Eip712,
+    Eip1271,
     EthSign,
     PreSign,
 }
-#[derive(Eq, PartialEq, Clone, Copy, Debug, Deserialize, Serialize, Hash)]
+
+#[derive(Eq, PartialEq, Clone, Debug, Deserialize, Serialize, Hash)]
 #[serde(rename_all = "lowercase")]
 #[serde(tag = "signingScheme", content = "signature")]
 pub enum Signature {
     Eip712(EcdsaSignature),
+    Eip1271(#[serde(with = "prefixed_hex")] Vec<u8>),
```

Comment:
Oh, great! I wasn't aware. I will change it asap.

Reply:
PS: One small issue I find is that it does not guard against an empty hex. Will this be a problem?

Reply:
TBH I don't know what the data would look like here (i.e. variable vs. constant length) but I think this should not be an issue.
Also if this is a normal hash you could also just use `primitive_types::H256` or something like that. This would also check that the string has the right length.

Reply:
> One small issue I find is that it does not guard against an empty hex

Hmm, it works for me as expected and allows empty bytes:
```rust
println!("{:?}", bytes_hex::deserialize(json!("0x"))); // Ok([])
```

Also, to my understanding, there is no rule with EIP-1271 that the signature requires `signature.length > 0`.

> TBH I don't know what the data would look like here (i.e. variable vs. constant length) but I think this should not be an issue.

It is a variable byte-length signature (i.e. its implementation defined what is in the signature).

Reply:
> Also, to my understanding, there is no rule with EIP-1271 that the signature requires signature.length > 0.
Then it makes sense to allow empty bytes.

And yes the signature is variable byte-length.